src/LasagnaController.cpp

```diff
@@ -1,28 +1,34 @@
 #include "LasagnaController.h"
+#include <cmath>
 
 LasagnaController::LasagnaController() :
   v1_filter(1./60./16., 0.5, 20.),
   v2_filter(1./60./7.,  0.5, 20.),
   action_filter(1./60./16., 0.5, 20.)
-{}
+{
+  calcGains();
+}
 
 LasagnaController::LasagnaController(float freq) : 
   v1_filter(1./60./16., 0.5, freq),
   v2_filter(1./60./7.,  0.5, freq),
   action_filter(1./60./16., 0.5, freq) 
 {
   constants.freq = freq;
+  calcGains();
 }
 
+
+
 bool LasagnaController::update(Input input){ 
 
   /**
    * Input protection for dtdl_ext
    */
-  if(isnan(input.dldt_ext)){
+  if(std::isnan(input.dldt_ext)){
     input.dldt_ext = 0; 
   } else {
-    float limit = 0.0001*4;
+    float limit = 0.6; // limit on what the max value for sunset dldt can be, in grams/s
```

Comment:
This seems like something we should be able to configure over rockblock

Reply:
We can already disable it over rockblock, this is just an aditional sanity check